src/filter/GaussianFilter.js

```diff
@@ -0,0 +1,58 @@
+import IJ from '../ij';
+
+export default function gaussian_filter(n, sigma){
```

Comment:
because we are in the directory "filter" we should just name this method "gaussian". The name of the file should also just be "gaussian.js"